R/tokenizer.R

```diff
@@ -34,7 +34,8 @@ NULL
 
 #' @export
 #' @rdname Tokenizers
-#' @param na String to use for missing values.
+#' @param na Character vector of strings to use for missing values. Set this
```

Comment:
I think it would be slightly simpler to use `character()` (i.e. a zero length character vector)` to indicate no missings

And maybe the default value for na should become `c("", "NA")`?